ArduCopter/mode.h

```diff
@@ -935,6 +937,14 @@ class ModeGuided : public Mode {
     void set_yaw_state(bool use_yaw, float yaw_cd, bool use_yaw_rate, float yaw_rate_cds, bool relative_angle);
     bool use_pilot_yaw(void) const;
 
+    // parameters
+    AP_Int32 options;       // options that can be applied to change guided mode behaviour
+    AP_Float timeout;       // timeout (in seconds) after which vehicle will stop or return to level if no updates are received from caller.  Only applicable during velocity, acceleration or angle control
+    AP_Float velxy_max;     // maximum horizontal velocity when using velocity or acceleration control
```

Comment:
I think velxy_max, velz_max, accelxy_max and accelz_max are not being used so we should remove them.